### PR TITLE
Update media delivery and ad selection calculations

### PR DESCRIPTION
**Pull request by EmmaLouise2018**

### CODE REVIEW

File: docs/calculations.mdx [hunk 1]
```diff
@@ -437,42 +437,55 @@ creative_consumer_device_embodied_emissions_gco2pm =
...
       media_data_transfer_embodied_emissions_gco2pm
+```
+
+Determing the session seconds for a single impressions
 
 ```
+session_seconds_per_imp =
+      property.average_seconds_per_session_ex_ads x
+      property.ad_funded_percentage /
+      property.average_imps_per_session
```
Comment (EmmaLouise2018) [inline docs/calculations.mdx:451]: @bokelley I think we need to add defaults for these values if we do not have them at the property level, likely broken out by channel

File: docs/calculations.mdx [hunk 2]
```diff
@@ -437,42 +437,55 @@ creative_consumer_device_embodied_emissions_gco2pm =
...
+```
 
 For all channels other than CTV/BVOD, use the conventional model:
 
 ```
+media_data_transfer_kb_per_imp =
+      property.average_data_kb_per_session_seconds_ex_ads x
```
Comment (EmmaLouise2018) [inline docs/calculations.mdx:458]: Think we need to add in a default for when we don't have property specific values. Likely broken out by channel

File: docs/calculations.mdx [hunk 3]
```diff
@@ -275,22 +275,22 @@ Providers should clearly and publicly document:
       average_dynamic_watts_per_mbps
 
   lookup_channel:
-    if property.channels.length == 1:
+      if property.channels.length == 1:
```
Comment (EmmaLouise2018) [inline docs/calculations.mdx:278]: Just formatting - we were using a mix of different size tabs

File: docs/calculations.mdx [hunk 4]
```diff
@@ -513,15 +526,16 @@ get_platform_emissions(ad_platform):
...
 
 ### Ad Selection - Data Transfer
 
 ```
 ad_selection_data_transfer_emissions_gco2pm =
-    data_transfer_bytes / 1024 x
-    lookup_carbon_intensity_gco2e_per_kwh(country, region, utc_datetime)
+      data_transfer_bytes / 1024 x
+      usage_kwh_per_gb / 1000000 x
+      lookup_carbon_intensity_gco2e_per_kwh(country, region, utc_datetime)
```
Comment (EmmaLouise2018) [inline docs/calculations.mdx:540]: I believe we were missing `usage_kwh_per_gb`

File: docs/calculations.mdx [hunk 5]
```diff
@@ -437,42 +437,55 @@ creative_consumer_device_embodied_emissions_gco2pm =
...
+      baseload_watts +
+      dynamic_watts_per_mbps x video_bitrate_kbps / 1000
+
 media_data_transfer_usage_emissions_gco2pm =
-    session_seconds_per_imp x
-    power x
-    lookup_carbon_intensity_gco2e_per_kwh(country, region, utc_datetime)
+      session_seconds_per_imp / 3600 x
+      power_watts / 1000 x
+      lookup_carbon_intensity_gco2e_per_kwh(country, region, utc_datetime)
```
Comment (EmmaLouise2018) [inline docs/calculations.mdx:483]: Missing unit conversions

File: docs/calculations.mdx [hunk 6]
```diff
@@ -437,42 +437,55 @@ creative_consumer_device_embodied_emissions_gco2pm =
...
 media_data_transfer_usage_emissions_gco2pm =
-      property.average_data_kb_per_session_ex_ads x
+      media_data_transfer_kb x
       usage_kwh_per_gb / 1000000 x
       lookup_carbon_intensity_gco2e_per_kwh(country, region, utc_datetime)
 
 media_data_transfer_embodied_emissions_gco2pm =
-      property.average_data_kb_per_session_ex_ads x
+      media_data_transfer_kb x
       embodied_emissions_gco2e_per_kb
```
Comment (EmmaLouise2018) [inline docs/calculations.mdx:468]: Should be a data transfer bytes/kb number here so broke out calculating `media_data_transfer_kb_per_imp` above to then use in these calculations

File: docs/calculations.mdx (hunk 1 above)
Comment (bokelley) [reply docs/calculations.mdx:451]: We have decent session time data from similarweb/data.ai so we can pull an 80th percentile number and contribute. Feels like 100% is the right ad funded default (to be conservative), and we can pull same imps per session numbers that we had before (I think this is really a "imps per session second" default like we have now?)

File: docs/calculations.mdx (hunk 2 above)
Comment (bokelley) [reply docs/calculations.mdx:458]: I think for CTV/audio this comes from the average bitrate, probably. For app-based channels let's pull an 80th percentile number based on what we have from data.ai... I think we could probably get web from a sampling of archive.org if we want, or can take 80th percentile of our database? This one would be helped with Sincera data prob.

**Review by bokelley — APPROVED** [review]

**Pull request merged**